sdv/utils/poc.py

```diff
@@ -139,3 +142,66 @@ def get_random_subset(data, metadata, main_table_name, num_rows, verbose=True):
 
     metadata.validate_data(result)
     return result
+
+
+def get_random_sequence_subset(
```

Comment:
Is it a `utils` or a `poc` method? The issue mentioned to import the method through `sdv.utils` which is different from `sdv.utils.poc`

Reply:
I'm not sure. The issue said to add to utils but was ambiguous as to whether that meant the module or package. @npatki Where should it be

Reply:
I'm not sure what the difference is @amontanez24? As long as the code snippet works then I am fine with whatever is done internally in the code.

```python
from sdv.utils import get_random_sequence_subset
```